### DISCUSSION TITLE
What do I have to do to install the v2 version?

### DISCUSSION THREAD

**Discussion by DOMGO-cut** [postalserver/postal#3075] [Development discussions]
curl https://raw.githubusercontent.com/postalserver/install/main/prerequisites/install-ubuntu.v2.sh | bash
What do I have to do to install the v2 version? I tried this line of v2 commands but the installation is 3.3.4...

**Comment by willpower232** [comment]
If you specifically need the v2 docker image, you would have to craft your own setup.

If you're able to use the v3 docker image, you can use https://raw.githubusercontent.com/postalserver/install/main/prerequisites/install-ubuntu.v3.sh

The only major difference is that v3 doesn't use RabbitMQ.

Is something still linking to the v2 script instead of the v3 script?
